sources/models/shared-defaults/aws-autoscaling.toml

```diff
@@ -0,0 +1,3 @@
+# Autoscaling warm pool support
+[settings.autoscaling]
```

Comment:
The name of this file is no longer correct, right? The setting was moved out of `aws` so all platforms want this default now?

Reply:
All `aws` variants should have this, but it's `settings.autoscaling`, not `settings.aws.autoscaling`. `vmware` and `metal` variants won't have this.